clang/lib/Basic/Targets/AArch64.cpp

```diff
@@ -741,7 +722,7 @@ bool AArch64TargetInfo::handleTargetFeatures(std::vector<std::string> &Features,
 
     if (Feature == "+neon" || Feature == "+fp-armv8")
       FPU |= NeonMode;
-    if (Feature == "+jscvt") {
+    if (Feature == "+jsconv") {
```

Comment:
This looks like bug fix which would make a functional difference, does it need a test?

Reply:
Agree, added a test case to `aarch64-target-features.c`